xray/commands/audit/generic/jas/applicabilityscanner.go

```diff
@@ -266,29 +261,29 @@ func (a *ApplicabilityScanManager) parseResults() error {
 
 	xrayCves := removeDuplicateValues(a.createCveList())
 	for _, xrayCve := range xrayCves {
-		a.applicabilityScannerResults[xrayCve] = "unknown"
+		a.applicabilityScannerResults[xrayCve] = UndeterminedStringValue
 	}
 
 	for _, vulnerability := range fullVulnerabilitiesList {
 		applicableVulnerabilityName := getVulnerabilityName(*vulnerability.RuleID)
 		if isVulnerabilityApplicable(vulnerability) {
-			a.applicabilityScannerResults[applicableVulnerabilityName] = "Yes"
+			a.applicabilityScannerResults[applicableVulnerabilityName] = ApplicableStringValue
 		} else {
-			a.applicabilityScannerResults[applicableVulnerabilityName] = "No"
+			a.applicabilityScannerResults[applicableVulnerabilityName] = NotApplicableStringValue
 		}
 	}
 	return nil
 }
 
 func (a *ApplicabilityScanManager) DeleteApplicabilityScanProcessFiles() error {
-	if _, err := os.Stat(a.configFileName); err == nil {
-		err = os.Remove(a.configFileName)
+	if exist, _ := fileutils.IsFileExists(a.configFileName, false); exist {
```

Comment:
I don't think we should return this error. this function is called as defer() whenever the Run() function is failing. Meaning it can be called before the config file/the results file have been created. what do you think?